kbase.yml

```diff
@@ -8,7 +8,7 @@ service-language:
     python
 
 module-version:
-    0.11.7
+    0.11.8
 
```

Comment:
0.11.7 isn't registered anywhere, there's no need for a version bump

Reply:
👍

Reply:
Release notes still have a version bump

Reply:
so I just add under [0.11.7]? But how do I update the corresponding date?

Reply:
yep. I'd just set the date to TBD for now then change it when you register

Reply:
👍